pkgs/kde/lib/mk-kde-derivation.nix

```diff
@@ -124,4 +124,4 @@ in
       position = "${pos.file}:${toString pos.line}";
     } // (args.meta or { });
   in
-    stdenv.mkDerivation (defaultArgs // cleanArgs) // { inherit meta; }
+    stdenv.mkDerivation (defaultArgs // cleanArgs // { inherit meta; })
```

Comment:
I don't think that is correct. To apply meta.pos correctly, it needs to be merged after mkDerivation got evaluated. When I do `nix edit kdePackages.ark` on this branch, this file gets opened, instead of pkgs/kde/gear/ark/default.nix like on nixos-unstable currently.

Reply:
please see https://github.com/NixOS/nixpkgs/pull/297780

Reply:
But doing it the other way around breaks `meta.broken`.